### PR TITLE
Add JSON superset

### PR DESCRIPTION
**Pull request by gibson042**
Paging @erights... trying to make sure this doesn't get lost.

### CODE REVIEW

File: stage-0-proposals.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Stage 0 proposals are either
 |   | [Function bind syntax][bind-syntax]                                        | Brian Terlson & Matthew Podwysocki   | 0     |
 |   | [Function expression decorators][func-expr-decorators]                     | Igor Minar                           | 0     |
 |   | [`is{Type}` methods][is-types]                                             | James M Snell                        | 0     |
+|   | [JSON superset][json-superset]                                             |                                      | 0     |
```
Comment (ljharb) [inline stage-0-proposals.md]: to be on the list, it would need identified champion(s); typically this would be done before making this PR.

**Comment by erights** [comment]
I volunteer to champion. Once there's a september agenda, I'd appreciate a reminder to put an item there for this.

**Comment by ljharb** [comment]
@erights already exists https://github.com/tc39/agendas/blob/master/2017/09.md

**Comment by erights** [comment]
Done

**Pull request merged**